fuzz.go

```diff
@@ -61,6 +63,33 @@ func NewWithSeed(seed int64) *Fuzzer {
 	return f
 }
 
+// NewFromGoFuzz is a helper function that enables using gofuzz (this
```

Comment:
This is an awesome idea!

I think we can come up with a better implementation, however--I'll make some suggestions.

Reply:
Sure, go ahead :-)